drv/psc-seq-server/src/main.rs

```diff
@@ -73,10 +73,6 @@
 //! - Wait some time for it to wake.
 //! - Start watching the fault signal again.
 //!
-//! We count the number of times we have had to reset a PSU in a row, and if the
```

Comment:
In case anyone's curious, I removed this because it was leftover and doesn't accurately describe the final behavior of the sequencer, which will attempt to resuscitate a PSU forever right now (at intervals).